pkg/providers/tinkerbell/hardware/translate.go

```diff
@@ -45,7 +45,7 @@ func Translate(reader MachineReader, writer MachineWriter, validator MachineVali
 	}
 
 	if err != nil {
-		return fmt.Errorf("read: %v", err)
+		return fmt.Errorf("hardware.csv is not correct: read: %v", err)
```

Comment:
Its not appropriate to talk about a CSV file at this point as this particular algorithm is agnostic to the input source.

```suggestion
		return fmt.Errorf("read: invalid hardware: %v", err)
```